debezium-core/pom.xml

```diff
@@ -80,6 +80,27 @@
             <artifactId>kafka-connect-avro-converter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I'm surprised to see this in _debezium-core_, I'd rather have expected them in the MySQL connector module?